### DISCUSSION TITLE
alertmanager not work

### DISCUSSION THREAD

**Discussion by 1998729** [jertel/elastalert2#1023] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/88186802/203460022-34800a40-3148-46da-be83-35e7c4aec0fd.png)

<img width="1789" alt="image" src="https://user-images.githubusercontent.com/88186802/203460111-f4288c67-9c0d-40de-b8c4-5fcce51dfc75.png">

**Comment by 1998729** [comment]
There is no output or error, but the number is indeed hit. How to debug?

**Comment by 1998729** [comment]
@jertel

**Comment by 1998729** [comment]
Sorry, I added debug

**Comment by jertel** [comment, marked as answer]
I don't understand the question. I also deleted one of your images since it had full domain information posted. For security reasons I recommend all potentially sensitive information be masked in images, or substituted in log posts.
